src/makeCalculator.js

```diff
@@ -38,6 +38,40 @@
  */
 function makeCalculator() {
   // write code here
+
+  const calculator = {
+    result: 0,
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add(result, number) {
+      return result + number;
```

Comment:
You should pass only the number here. And seems that return is redundant here
```suggestion
    add(number) {
      this.result += number;
```